src/applications/static-pages/events/components/Results/ResultsWhereContent.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const ResultsWhereContent = ({
+  fieldFacilityLocation,
+  fieldLocationType,
+  locations,
+}) => {
+  function whereContent() {
+    if (fieldLocationType === 'online') return 'This is an online event.';
+    return (
+      <>
+        <a href={fieldFacilityLocation?.entity?.entityUrl.path}>
+          {fieldFacilityLocation?.entity?.title}
+        </a>
+        {locations?.length > 0 && (
+          <div>
+            {locations?.map(location => (
+              <p className="vads-u-margin--0" key={location}>
+                {location}
+              </p>
+            ))}
+          </div>
+        )}
+      </>
+    );
+  }
+
+  if (
+    fieldFacilityLocation === null &&
+    fieldLocationType !== 'online' &&
+    locations.length === 0
+  )
+    return <></>;
+
+  return (
+    <div className="vads-u-display--flex vads-u-flex-direction--row vads-u-margin-top--1">
+      <p className="vads-u-margin--0 vads-u-margin-right--0p5">
+        <strong>Where:</strong>
+      </p>
+      <div className="vads-u-display--flex vads-u-flex-direction--column">
+        <p className="vads-u-margin--0">{whereContent()}</p>
```

Comment:
This would become:

```
<WhereContent 
   fieldFacilityLocation={fieldFacilityLocation}
   fieldLocationType={fieldLocationType}
   locations={locations}
/>
```